mender-dist-packages-building/armhf/Dockerfile.rpi

```diff
@@ -0,0 +1,29 @@
+FROM scratch
+ADD output/rootfs.tar.xz /
+ARG SUITE
+
+RUN debootstrap/debootstrap --second-stage
+
+ENV GOARCH=arm
+ENV DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update && \
+    apt-get install -y \
+    build-essential \
+    dh-make \
+    git \
+    wget \
+    curl \
+    debhelper \
+    devscripts \
+    equivs \
+    liblzma-dev \
+    pkg-config \
+    libssl-dev \
+    libsystemd-dev \
+    cmake && \
```

Comment:
One more thing,

This list of packages is supposed to be the same one that for the "normal" images. Can we have it on an external file so that both can use it? Something like [this](https://github.com/mendersoftware/mender-artifact/blob/ff51653598f1365a31e7732543cd2db1e3d11e21/deb-requirements.txt) + [this](https://github.com/mendersoftware/mender-artifact/blob/ff51653598f1365a31e7732543cd2db1e3d11e21/Makefile#L105).